### PR TITLE
added missing mob IDs

### PR DESCRIPTION
**Pull request by JayTDawgzone**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [ ] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [ ] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by CatsEyeXI** [comment]
@JayTDawgzone since you prefer to do it this way (hardcode the mob ID's into the zone ID.lua), I suggest we use DYNAMIC_LOOKUP to shift proof your work... 

`scripts/zones/Arrapago_Reef/IDs.lua`

is a good example, see how Medusa is set up as a dynamic lookup.

File: scripts/zones/Labyrinth_of_Onzozo/IDs.lua [hunk 1]
```diff
@@ -83,6 +83,7 @@ zones[xi.zone.LABYRINTH_OF_ONZOZO] =
             [17649773] = 17649761, -- -65.089 5.386 81.363
             [17649766] = 17649761, -- -64.269 5.441 72.382
         },
+        LORD_OF_ONZOZO          = 17093078,
```
Comment (CatsEyeXI) [inline scripts/zones/Labyrinth_of_Onzozo/IDs.lua:86]: Convert to dynamic lookup

File: scripts/zones/Mamook/IDs.lua [hunk 2]
```diff
@@ -44,6 +44,7 @@ zones[xi.zone.MAMOOK] =
             [17043773]           = 17043779, -- -201.522 17.209 -363.865
             [17043774]           = 17043779, -- -206.458 17.525 -373.798
         },
+        HUNDREDFACED_HAPOOL_JA   = 17043665,
```
Comment (CatsEyeXI) [inline scripts/zones/Mamook/IDs.lua:47]: Convert to Dynamic Lookup

File: scripts/zones/RoMaeve/IDs.lua [hunk 3]
```diff
@@ -48,6 +48,7 @@ zones[xi.zone.ROMAEVE] =
             [17277075] = 17277079,
             [17277078] = 17277079,
         },
+        SHIKIGAMI_WEAPON  = 17277127,
```
Comment (CatsEyeXI) [inline scripts/zones/RoMaeve/IDs.lua:51]: Convert to Dynamic Lookup

File: scripts/zones/Wajaom_Woodlands/IDs.lua [hunk 4]
```diff
@@ -52,6 +52,7 @@ zones[xi.zone.WAJAOM_WOODLANDS] =
             [16986195] = 16986197, -- 219.000 -18.000 -59.000
             [16986196] = 16986197, -- 203.000 -16.000 -74.000
         },
+        HYDRA                  = 16986355,
```
Comment (CatsEyeXI) [inline scripts/zones/Wajaom_Woodlands/IDs.lua:55]: Convert to Dynamic Lookup

**Review by CatsEyeXI — CHANGES_REQUESTED** [review]

**Comment by CatsEyeXI** [comment]
also, the pull request template contains 3 checkboxes, please make sure you complete and mark them complete .

image

**Comment by JayTDawgzone** [comment]
Will do, I was going to make a data table in the module to remove any dependency from the scripts, but ended up doing this on Hax's advice. It appears that I have some more work to do in the database tables in modules anyways, so I'll close this for now and come back to it.

**Pull request closed without merging**